Zend/zend_closures.c

```diff
@@ -104,7 +104,7 @@ static zend_bool zend_valid_closure_binding(
 	}
 
 	if (is_fake_closure && scope != func->common.scope) {
-		zend_error(E_WARNING, "Cannot rebind scope of closure created by ReflectionFunctionAbstract::getClosure()");
+		zend_error(E_WARNING, "Cannot rebind scope of closure");
```

Comment:
I think this just shifts confusing in the other direction. Maybe `Cannot rebind scope of closure created from method` or so?

Reply:
Well, in this case the closure has been created from a free standing function. I wonder whether we can check that with `func->common.scope == NULL`, and use a separate error message in this case.

Reply:
Sounds reasonable.